src/components/BookForm.js

```diff
@@ -16,21 +22,42 @@ const AddBook = () => {
     e.preventDefault();
     const eventCategory = document.getElementById('category');
     const bookcategory = eventCategory.options[eventCategory.selectedIndex].text;
-    const newBook = { title: booktitle, category: bookcategory, item_id: uuidv4() };
+    const newBook = {
+      title: booktitle, category: bookcategory, item_id: uuidv4(), author: bookauthor,
+    };
     dispatch(addBook(newBook));
     setTitle('');
+    setAuthor('');
+  };
+
+  const myStyle = {
+    fontSize: '1rem',
+    fontFamily: ['Montserrat', 'sans-serif'],
+  };
+
+  const myStyle2 = {
+    width: '30%',
+    height: '2.813rem',
+    padding: '1rem',
+    borderRadius: '4px',
+    border: 'solid 1px white',
+    backgroundColor: '#fff',
   };
 
   return (
-    <div className="mx-auto m-4">
-      <h2>ADD NEW BOOK</h2>
+    <div className="card-style mx-auto m-4">
+      <h2 style={myStyle} className="fw-bold text-secondary">ADD NEW BOOK</h2>
+
       <form onSubmit={submitBookToStore}>
-        <input type="text" id="titleId" value={booktitle} onChange={addTitle} placeholder="Book title" />
-        <select type="select" id="category">
-          <option value="test_1">Learning</option>
-          <option value="test_2">Programing</option>
-          <option value="test_3">Movies</option>
-          <option value="test_4">Novels</option>
+        <input type="text" id="titleId" value={booktitle} onChange={addTitle} style={myStyle2} placeholder="Book title" />
+        <input type="text" id="authorId" value={bookauthor} onChange={addAuthor} style={myStyle2} placeholder="Book Author" />
+        <select type="select" className="p-0 w-25" id="category" style={myStyle2}>
+          <option value="Sci-Fi">Sci-fi</option>
+          <option value="Learning">Learning</option>
+          <option value="Programming">Programming</option>
+          <option value="Drama">Drama</option>
+          <option value="Thriller">Thriller</option>
+          <option value="Action">Action</option>
```

Comment:
- [ ] There should be a `<hr>` element between `BookList` and `BookForm`, please add it.
- [ ] Kindly make sure to update the styling of your form (button and input fields) to match the given design.
- [ ] The `author` name is found in your form but not in each `book`, kindly update it (I would also suggest removing the author input field while keeping the author's name in each `book` to match the design). 

Zeplin design
![image](https://user-images.githubusercontent.com/33767951/142712135-4582a78d-ef36-456b-ae53-71108220fff8.png)

Your Styling
![image](https://user-images.githubusercontent.com/33767951/142712148-1ce4638f-bf92-4722-89f0-b8d75a2a539e.png)